astro/src/components/quickstarts/DockerRequirements.astro

```diff
@@ -0,0 +1,23 @@
+---
+import Code from 'astro/components/Code.astro';
+import Aside from 'src/components/Aside.astro';
+---
+<dd>
+<h2> Docker Requirements</h2>
+<Aside type="note">
+    You need to have Docker and Docker Compose installed.</Aside>
+
+
+<p>This document has been tested with Docker version 20, but should work with other versions.</p>
+
+  <i>Testing the Docker Installation</i>
+
+ <Code lang="shell" code="docker compose up -d" />
```

Comment:
This is the wrong command. At least, it didn't work for me.

Reply:
Fixed